src/scripts/main.js

```diff
@@ -1,3 +1,305 @@
+    form.append(newElement);
+
+    return;
+  };
+
+  label.textContent = text;
+
+  form.append(label);
+  label.append(newElement);
+};
+
+createInForm('input', 'Name:');
+createInForm('input', 'Position:');
+createInForm('select', 'Office:');
+createInForm('input', 'Age:');
+createInForm('input', 'Salary:');
+createInForm('button', 'Save to table');
+
+const addDataQa = (element, value) =>
+  element.setAttribute('data-qa', `${value}`);
+
+const nameOfThePerson = form.children[0].firstElementChild;
+const position = form.children[1].firstElementChild;
+const select = form.children[2].firstElementChild;
+const age = form.children[3].firstElementChild;
+const salary = form.children[4].firstElementChild;
+const button = form.lastElementChild;
+
+addDataQa(nameOfThePerson, 'name');
+addDataQa(position, 'position');
+addDataQa(select, 'office');
+addDataQa(age, 'age');
+addDataQa(salary, 'salary');
+
+age.type = 'number';
+salary.type = 'number';
+
+const createOption = (text) => {
+  const option = document.createElement('option');
+
+  option.append(text);
+  select.append(option);
+};
+
+createOption('Tokyo');
+createOption('Singapore');
+createOption('London');
+createOption('New York');
+createOption('Edinburgh');
+createOption('San Francisco');
+
+const createCell = () => document.createElement('td');
+
+const addToNewCell = (newCell, formInput) => {
+  newCell.textContent = formInput;
+};
+
+button.addEventListener('click', (e) => {
+  const newLine = document.createElement('tr');
+
+  const employeeName = form.children[0].firstElementChild;
+  const employeePosition = form.children[1].firstElementChild;
+  const employeeOffice = form.children[2].firstElementChild;
+  const employeeAge = form.children[3].firstElementChild;
+  const employeeSalary = form.children[4].firstElementChild;
+
+  e.preventDefault();
+
+  newLine.append(createCell());
+  newLine.append(createCell());
+  newLine.append(createCell());
+  newLine.append(createCell());
+  newLine.append(createCell());
+
+  addToNewCell(newLine.children[0], employeeName.value);
+  addToNewCell(newLine.children[1], employeePosition.value);
+  addToNewCell(newLine.children[2], employeeOffice.value);
+  addToNewCell(newLine.children[3], employeeAge.value);
+
+  newLine.children[4].textContent = '$'
+  + (+employeeSalary.value).toLocaleString();
+
+  if (employeeName.value.length < 4) {
+    pushNotification(630, 10, 'Ooops, we\'ve got a trouble',
+      'Please, fill in at least 4 letters to add a new employee', 'error');
+
+    return;
+  }
+
+  if (!position.value) {
+    pushNotification(630, 10, 'Ooops, we\'ve got a trouble',
+      'Please, add an employee\'s position ', 'error');
+
+    return;
+  }
+
+  if (employeeAge.value < 18 || employeeAge.value > 90) {
+    pushNotification(630, 10, 'Ooops, we\'ve got a trouble',
+      'Please, note that employee\'s age has got to be between '
+      + '18 and 90 range. Give it another try!', 'error');
+
+    return;
+  }
+
```

Comment:
clear form after submitting:

![image](https://user-images.githubusercontent.com/86923089/200171445-9908dc25-f8ea-4dc4-9cf6-1153a62efc62.png)

also (not critical) I'd make salary field required